openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiMessages.java

```diff
@@ -15,19 +15,22 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 package io.ballerina.openapi.cmd;
 
 /**
  * This class contains the messages constants required for OpenApi tool.
+ *
+ * @since 1.3.0
  */
-public class OpenApiMesseges {
+public class OpenApiMessages {
```

Comment:
Shouldn't this be renamed as `ErrorMessages`?

Reply:
Yeah we can use as `ErrorMessages` will re change it :)